cudapoa/src/cudapoa_batch.hpp

```diff
@@ -66,6 +66,9 @@ class CudapoaBatch : public Batch
     StatusType get_msa(std::vector<std::vector<std::string>>& msa,
                        std::vector<StatusType>& output_status);
 
+    void get_graphs(std::vector<DirectedGraph>& graphs,
```

Comment:
Is there really added value in getting the status for each individual graph retrieval instead of just returning one status?

Reply:
Some of the POAs don't get generated, and sometimes useful to know what the error is.

Reply:
ok, makes sense.